analyzer-core/src/main/java/ffb/analyzer/core/queue/PriorityTask.java

```diff
@@ -0,0 +1,32 @@
+package ffb.analyzer.core.queue;
+
+
+
+public class Task implements Comparable<Task> {
+
+    public Long id;
+    public int priority;
+
+
+    @Override
+    public int compareTo(Task otherTask) {
+        return this.getId().compareTo(otherTask.getId());
+    }
+
+
+    public int getPriority() {
+        return priority;
+    }
+
+    public void setPriority(int priority) {
+        this.priority = priority;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
```

Comment:
This shouldn't be exposed as a public method. We shouldn't be able to changed the ID.